### ISSUE TITLE
End user friendly Web UI...

### ISSUE THREAD

**Issue by oxcxsxixbxaxn** [xoseperez/espurna#699]
ESPurna does exactly what I want in terms of RBG and white LED control and it is very configurable but I would like to hve a little more control over the Web UI to simplify it for every day users.
I would quite like to have simpler user interface with just the controls and none of the config, so log in for admin, but none or perhaps user specific login to access just the controls and maybe the schedule.

Would also be nice to be able to change labels for those users who have no interest in channel numbers.
eg change "brightness" / "channel 4" to "colour dimmer" / "white dimmer"
![image](https://user-images.githubusercontent.com/9394156/37648082-d56d0108-2c25-11e8-93c4-b5916e34b689.png)

**Comment by mcspr** [comment]
Yeah, I feel like it would be nice to jump directly to color panel without loading the first page first.

i.e. having separate urls for every panel
http://espurna.lan/status
http://espurna.lan/ntp
http://espurna.lan/color
...

I remember doing something similar for personal project using [web history api](https://developer.mozilla.org/en/docs/Web/API/History) and `replaceState` but it is a bit fuzzy
